horace_core/sqw/coord_transform/@ortho_proj/ortho_proj.m

```diff
@@ -334,7 +347,49 @@
             % property
             [~, mat] = obj.uv_to_rot();
             mat = [mat,[0;0;0];[0,0,0,1]];
-        end        
+        end
+
+        function  [rlu_to_ustep, u_rot, ulen] = uv_to_rot(proj,ustep)
+            % Determine the matrices partially used for conversion
```

Comment:
what does partially mean here? Please describe the partialness and point to it in the code

Also, why have you moved this function from after the next one? It means I can't see change in the code.

Reply:
I do not remember what "partial" meant in this case (may be 3x3 or only some of them used) so removed this word.